README.md

```diff
@@ -100,7 +104,33 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+
+  attr_accessor :name
+
+  def initialize(name)
+    @name = name
+  end
+
+  def greet
+    "Hi, my name is #{name.capitalize}."
+  end
+
+end
+
+pumba = Animal.new("Pumba")
+
+class Lion < Animal
+  def initialize(name, pack, king)
+    super(name)
+    @pack = pack
```

Comment:
`pack` should be a class variable, meaning it exists independently of an instance and should be defined with `@@`

Also, `@@pack` should contain references to each Lion instance created. So inside of your Lion's `initialize` method, you could do something like this...

```rb
class Lion < Animal
  @@pack = []

  def initialize
    @@pack << self # where self = instance being created
  end
end
```